portworx/templates/portworx-storagecluster.yaml

```diff
@@ -24,14 +25,17 @@ spec:
       value: {{ $serviceAccount }}
     {{- end }}
     - name: PX_IMAGE
-      value: "{{template "px.getImage" . }}"
+      value: docker.io/portworx/px-enterprise:2.12.0
```

Comment:
Why are we hard-coding value of PX-Enterprise image here?

Reply:
Ideally this image should also be pulled from the same custom registry like the other, but with ECR registry it is facing an authentication issue, where the OCI-mon pod doesn't have enough access to pull the image. Me and Tapas were working on finding a fix for this. We did find one way where we could provide the access key through the registrySecret field, but the solution doesn't work in a production flow, because it is not possible for someone outside the org to follow it.

For the time being, Hard-coding the complete path seemed to fix the issue. With it we wanted to make the degraded service available back again until we find a better solution

Reply:
I have added the comment, and created a PWX ticket:- https://portworx.atlassian.net/browse/PWX-27961